--- mpi/longlong.h.orig	2011-11-20 11:55:14.000000000 +0900
+++ mpi/longlong.h	2011-11-20 11:54:14.000000000 +0900
@@ -225,8 +225,8 @@
 	   : "r0", "r1", "r2")
 #else
 #define umul_ppmm(xh, xl, a, b)                                         \
-  __asm__ ("%@ Inlined umul_ppmm\n"                                     \
-	   "umull %r1, %r0, %r2, %r3"                                   \
+  __asm__ ("@ Inlined umul_ppmm\n"                                     \
+	   "umull r1, r0, r2, r3"                                   \
 		   : "=&r" ((USItype)(xh)),                             \
 		     "=r" ((USItype)(xl))                               \
 		   : "r" ((USItype)(a)),                                \
